wix-embedded-mysql/src/main/java/com/wix/mysql/config/MysqldConfig.java

```diff
@@ -100,6 +101,11 @@ public static Builder aMysqldConfig(final Version version) {
         return new Builder(version);
     }
 
+    @Override
```

Comment:
Is this related to fix you are making? If so, then I would like to have a test for this. If not lets just drop it:)

Reply:
👍